project/Dependencies.scala

```diff
@@ -36,9 +36,9 @@ object Dependencies {
     val pubsubFs2          = "0.12.0"
     val httpClient         = "0.20.0"
     val logging            = "0.3.0"
-    val scio               = "0.8.1"
```

Comment:
Can you double-check that there's no `Set` used any bad rows or other types flowing in `PCollection`?. Spotify made it non-deterministic in 0.8.2 https://github.com/spotify/scio/pull/2674

Reply:
AFAICT, we're not using `Set` in a place that can cause issues.

Reply:
To be more precise: We don't use `Set` in bad rows; and in the BQ Loader itself, we use it in an `SCollection` here: https://github.com/snowplow-incubator/snowplow-bigquery-loader/blob/1d1a5b66913cd86f9c9eab24323770ab9d742fb0/loader/src/main/scala/com/snowplowanalytics/snowplow/storage/bigquery/loader/Loader.scala#L122 but not to `groupBy`.

Do you think it's still best to change it just in case?

Reply:
No, I think without groupBy it should be ok.